kuttl-test.yaml

```diff
@@ -0,0 +1,25 @@
+#
+# EXECUTION (from install_yamls repo root):
+#
+#   make neutron_kuttl
+#
+# ASSUMPTIONS:
+#
+# 1. Latest version of kuttl is installed at /usr/local/bin/kubectl-kuttl
+#    - wget https://github.com/kudobuilder/kuttl/releases/download/v0.11.1/kubectl-kuttl_0.11.1_linux_x86_64
+#    - mv kubectl-kuttl_0.11.1_linux_x86_64 /usr/local/bin/kubectl-kuttl
+#    - chmod 755 /usr/local/bin/kubectl-kuttl
+# 2. An OCP 4.10+ CRC cluster with Podified Operators has been deployed
+# 3. CLI user has access to $KUBECONFIG
+# 4. The environment variable INSTALL_YAMLS is set to the the path of the
+#    install_yamls repo
+
+apiVersion: kuttl.dev/v1beta1
+kind: TestSuite
+reportFormat: JSON
+reportName: kuttl-test-neutron
+namespace: openstack
+timeout: 180
```

Comment:
adding link incase useful for other reviewers

these config attributes are described there https://kuttl.dev/docs/testing/reference.html#testsuite
e.g. timeout is in seconds

Reply:
though perhaps the subject of the pull request can be updated

it is the same as the install_yamls one https://github.com/openstack-k8s-operators/install_yamls/pull/99 where it makes more sense

Reply:
Ah, you're right, I've mixed them up, since I created one after the other, I'll update it, thanks